src/parser/MaintainSentences.h

```diff
@@ -76,11 +59,123 @@ class ColumnSpecificationList final {
 };
 
 
+class SchemaPropItem final {
+public:
+    using Value = boost::variant<int64_t, bool, std::string>;
+
+    enum PropType : uint8_t {
+        TTL_DURATION,
+        TTL_COL,
+    };
+
+    SchemaPropItem(PropType op, int64_t val) {
+        propType_ = op;
+        propValue_ = val;
+    }
+
+    SchemaPropItem(PropType op, bool val) {
+        propType_ = op;
+        propValue_ = val;
+    }
+
+    SchemaPropItem(PropType op, std::string val) {
+        propType_ = op;
+        propValue_ = std::move(val);
+    }
+
+    int64_t asInt() {
```

Comment:
If you don't want to expose the methods to outside, just make them private.

Reply:
👍，good catch!